core-rust/build.gradle

```diff
@@ -104,18 +104,26 @@ task buildRustDebug(type: Exec) {
 
 task buildRustForDocker(type: Exec) {
     def target = "x86_64-unknown-linux-gnu"
-
+    def docker_plattform = "linux/amd64"
```

Comment:
Minor typo: `docker_platform`* but also to be consistent with the other variable names, we should probably case it as `dockerPlatform`